challenge_6.js

```diff
@@ -6,11 +6,41 @@
  */
 exports.SumOfASquare = SumOfASquare;
 
-function SumOfASquare(naturalNumbers){
+function SumOfASquare(naturalNumbers) {
   this.naturalNumbers = naturalNumbers;
+
+  this.sumOfSquares = function () {
+    let result = 0;
+    let squared = 0;
+
+    for (let i = 0; i <= this.naturalNumbers; i++){
+      squared = i*i;
```

Comment:
You really don't need the `squared` variable here since it's really only being used once. It's best to just combine line 17 and 18 together by not using the variable at all.